packages/react/src/lib/VoiceProvider.tsx

```diff
@@ -63,30 +49,34 @@ export type VoiceContextType = {
   isAudioMuted: boolean;
   isPlaying: boolean;
   messages: (
-    | UserTranscriptMessage
-    | AssistantTranscriptMessage
+    | Hume.empathicVoice.UserMessage
+    | Hume.empathicVoice.AssistantMessage
     | ConnectionMessage
-    | UserInterruptionMessage
-    | JSONErrorMessage
-    | ToolCall
-    | ToolResponse
-    | ToolError
-    | ChatMetadataMessage
+    | Hume.empathicVoice.UserInterruption
+    | Hume.empathicVoice.WebSocketError
+    | Hume.empathicVoice.ToolCallMessage
+    | Hume.empathicVoice.ToolResponseMessage
+    | Hume.empathicVoice.ToolErrorMessage
+    | Hume.empathicVoice.ChatMetadata
```

Comment:
meeting note: If we ever have a new message type generated by Fern we will need to manually add it here in the React SDK. This is not ideal, but also not worse than the current flow today. We may return to this to see if it can be made less error prone (e.g. forgetting to add the type here when updating the `hume` package dependency version)